activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -355,12 +355,18 @@ def accepts_nested_attributes_for(*attr_names)
         options.update(attr_names.extract_options!)
         options.assert_valid_keys(:allow_destroy, :reject_if, :limit, :update_only)
         options[:reject_if] = REJECT_ALL_BLANK_PROC if options[:reject_if] == :all_blank
+        options[:klass] = self
```

Comment:
Could this assignment occur in the same Hash literal as `options = { allow_destroy: false, update_only: false }`?

```ruby
 options = { allow_destroy: false, update_only: false, klass: self }
```

Reply:
Can do, I just thought it felt more relevant after `options.assert_valid_keys`; so it's more like an internal option that a user provided one.

Also, if it comes after `options.update`, it's not overridable via an arg.

Reply:
On second thought, I might leave it as is for the reasons I pointed out.